app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -30,6 +32,8 @@
  */
 public class GameView extends SurfaceView implements Runnable {
 
+    private int currentLevel;
```

Comment:
nit: it's better to group all private int variables together, so maybe move this below?

Reply:
Fixed.